Code/Framework/AzToolsFramework/AzToolsFramework/UI/EditorEntityUi/EditorEntityUiHandlerBase.cpp

```diff
@@ -120,6 +120,7 @@ namespace AzToolsFramework
 
     bool EditorEntityUiHandlerBase::OnOutlinerItemDoubleClick([[maybe_unused]] const QModelIndex& index) const
     {
+        AzToolsFramework::EditorRequestBus::Broadcast(&AzToolsFramework::EditorRequestBus::Events::GoToSelectedEntitiesInViewports);
```

Comment:
The base Ui Handler is a stub class that is inherited from by other UI handlers, so as such it should not have behavior.

This code should be moved elsewhere. Ideally you would create another handler in-between the base and the prefab ui handler if you want this behavior to also be inherited by that.

Reply:
Indeed I see your point! I modified the code based on your suggestion

Reply:
Sounds better to me :) Hypothetically, we could also put it in the `else` branch of this if:
```
if (AZ::EntityId entityId = GetEntityIdFromIndex(index); auto entityUiHandler = m_editorEntityUiInterface->GetHandler(entityId))
```
this way handlers overriding that function would not get that behavior.
I think it makes sense to always apply it for now though, and we can make the change if any use-case arises where that is not what we want (but we would have to call that function manually in the Prefab handler then).

TLDR, thanks for changing this, looks good :)